backend/services/itinerary-service/internal/db/post-db.go

```diff
@@ -0,0 +1,188 @@
+package DBmodels
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/lib/pq"
+)
+
+type Post struct {
+	PostId       int       `json:"postId"`
+	ItineraryId  int       `json:"itineraryId"`
+	CreationDate time.Time `json:"dateOfCreation"`
+	Username     string    `json:"username"`
+	Tags         []string  `json:"tags"`
```

Comment:
If tags are stored in the itinerary do we need them here as well?

Reply:
Good Point. We don't. I'll remove those in the post.